packages/server/graphql/private/typeDefs/Mutation.graphql

```diff
@@ -360,6 +360,20 @@ type Mutation {
 		subscriptionId: ID!
 	): Boolean
 
+	"""
+	Called on stripe.invoice.payment_succeeded webhook to move an org to the team tier
+	"""
+	upgradeToTeamTier(
```

Comment:
The resolver was in private, so calling the mutation publicly wouldn't have worked anyways.